### PR TITLE
Fixes#87

### PR DESCRIPTION
**Pull request by RudiKlassen**

### CODE REVIEW

File: dashy/src/app/dashboard/shared/widget.service.spec.ts [hunk 1]
```diff
@@ -161,6 +161,7 @@ describe('Service: Widget', () => {
     inject([WidgetService], (service: WidgetService) => {
 
       let widgetConfig = new WidgetConfig('platform-status', 'dev', '/mock/data');
+      widgetConfig.titleUrl = 'http://www.foo.bar';
```
Comment (matthiasbalke) [inline dashy/src/app/dashboard/shared/widget.service.spec.ts]: do you need the titleUrl for this test case?

File: dashy/src/app/dashboard/shared/widget.service.spec.ts [hunk 2]
```diff
@@ -171,6 +172,7 @@ describe('Service: Widget', () => {
     inject([WidgetService], (service: WidgetService) => {
 
       let widgetConfig = new WidgetConfig('platform-status', 'dev', '/mock/data');
+      widgetConfig.description = 'http://www.foo.bar';
```
Comment (matthiasbalke) [inline dashy/src/app/dashboard/shared/widget.service.spec.ts]: do you need the description here? And why is the description an url?

File: dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts [hunk 3]
```diff
@@ -11,19 +12,28 @@ import {WidgetService} from '../../shared/widget.service';
 export class PlatformStatusWidgetComponent extends Widget {
 
   version: string = 'N/A';
+  titleUrl: string;
+  description: string;
```
Comment (matthiasbalke) [inline dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts]: `titleUrl` and `description` should be placed in `Widget` super class, as every widget has these attributes.

File: dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts [hunk 4]
```diff
@@ -11,19 +12,28 @@ import {WidgetService} from '../../shared/widget.service';
...
   status: Status = Status.DOWN;
 
   public constructor(private wS: WidgetService) {
     super(wS);
   }
 
+  public initWidget(widgetConfig: WidgetConfig) {
+    super.initWidget(widgetConfig);
+    this.titleUrl = widgetConfig.titleUrl;
+    this.description = widgetConfig.description;
```
Comment (matthiasbalke) [inline dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts]: same here. should be placed in super class

File: dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts [hunk 5]
```diff
@@ -11,19 +12,28 @@ import {WidgetService} from '../../shared/widget.service';
...
+
+    console.log(widgetConfig);
```
Comment (matthiasbalke) [inline dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts]: remove the debug code.

File: doc/examples/platformVersion/widget.platform-prod.json [hunk 6]
```diff
@@ -1,6 +1,8 @@
 {
   "title": "prod",
   "type": "platform-status",
+  "titleUrl": "http://putsreq.com/YH3ItymF2wFhiAi6ADD1",
+  "description": "Testplattform",
```
Comment (matthiasbalke) [inline doc/examples/platformVersion/widget.platform-prod.json]: typo: please use english examples

File: quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/rest/WidgetConfigResourceAssembler.java [hunk 7]
```diff
@@ -10,9 +13,6 @@
 import com.github.reflectoring.infiniboard.packrat.widget.WidgetConfig;
 import com.github.reflectoring.infiniboard.quartermaster.dashboard.rest.DashboardController;
 
-import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
```
Comment (matthiasbalke) [inline quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/rest/WidgetConfigResourceAssembler.java]: why did the import order change? Do we have to fix the code formatter?

File: dashy/src/app/dashboard/shared/widget-config.spec.ts [hunk 8]
```diff
@@ -2,18 +2,25 @@ import {WidgetConfig} from './widget-config';
...
     let type = 'platform-status';
     let widgetConfig = new WidgetConfig(type, 'Test Platform', '/mock/data');
+    widgetConfig.titleUrl = 'http://www.foo.bar';
     expect(widgetConfig.type).toEqual(type);
   });
 
   it('constructor() should set widgetConfig.title', () => {
     let title = 'Test Platform';
     let widgetConfig = new WidgetConfig('platform-status', title, '/mock/data');
+    widgetConfig.description = 'Dev Platform';
```
Comment (RudiKlassen) [inline dashy/src/app/dashboard/shared/widget-config.spec.ts]: I have deleted that line

File: dashy/src/app/dashboard/shared/widget.service.spec.ts (hunk 1 above)
Comment (RudiKlassen) [inline dashy/src/app/dashboard/shared/widget.service.spec.ts]: nope, i don´t need that  here. I´ve delted this line

File: dashy/src/app/dashboard/shared/widget.service.spec.ts (hunk 2 above)
Comment (RudiKlassen) [inline dashy/src/app/dashboard/shared/widget.service.spec.ts]: nope, i don´t need that  here. I´ve delted this line

File: dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts (hunk 3 above)
Comment (RudiKlassen) [inline dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts]: moved it

File: dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts (hunk 5 above)
Comment (RudiKlassen) [inline dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts]: check

File: doc/examples/platformVersion/widget.platform-prod.json (hunk 6 above)
Comment (RudiKlassen) [inline doc/examples/platformVersion/widget.platform-prod.json]: it is english now

File: quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/rest/WidgetConfigResourceAssembler.java (hunk 7 above)
Comment (RudiKlassen) [inline quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/rest/WidgetConfigResourceAssembler.java]: I didn´t change my formatter setting. Lets check it in our next meeting.

**Comment by matthiasbalke** [comment]
Which PR is the right one? PR #141 or #139? Please close one.

**Comment by matthiasbalke** [comment]
closing in favour of the newer one in #141.

**Pull request closed without merging**